edm4hep.yaml

```diff
@@ -809,6 +825,33 @@ datatypes:
       void {name}::setRec(const edm4hep::ReconstructedParticle& rec) { setTo(rec); }\n
       "
 
+  edm4hep::DriftChamberDigiSimTrackerHitLink:
+    Description: "Link between a DriftChamberDigi and a SimTrackerHit"
+    Author: "EDM4hep authors"
+    Members:
+      - float weight                      // weight of this link
+    OneToOneRelations:
+     - edm4hep::DriftChamberDigi from     // reference to the DriftChamberDigi
+     - edm4hep::SimTrackerHit to          // reference to the SimTrackerHit
+    ExtraCode:
+      declaration: "
+      [[deprecated(\"use getFrom instead\")]] edm4hep::DriftChamberDigi getRec() const;\n
```

Comment:
Why are there already deprecated functions added to a new datatype?

Reply:
They were there in case people wanted to use links in the "Association" way. But maybe we should not let that happen since there is no existing code using this data type. Removed these helpers.